courses/machine_learning/images/mnist_linear.ipynb

```diff
@@ -9,7 +9,12 @@
    "source": [
```

Comment:
<ul><li>move all imports at the beginning with the other "import tensorflow as tf" statement</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.